### PR TITLE
v2.2

### PR DESCRIPTION
**Pull request by francescoinfante**
- [x] add neo4jsearch
- [x] add link to neo2es for neo4j 3.3
- [x] guest mode

### CODE REVIEW

**Comment by francescoinfante** [comment]
ready to review

File: content/admin/content/access/guest-mode.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+By default, the guest mode is disabled.
+To enable it, set to `true` the `guestMode` configuration key inside the `access` configuration.
+
+A new user will appear in the [user list](/users/#creating-users) with the email `guest@linkurio.us`.
+This special user can only access the workspace and it can't save any visualization. Its access rights
```
Comment (davidrapin) [inline content/admin/content/access/guest-mode.md]: This special user can only access the `guest` page (`http://localhost:3000/guest`)

**Comment by davidrapin** [comment]
before merging:
- [ ] check that the neo4j DAO config doc is correct (Bolt)

**Comment by francescoinfante** [comment]
I think this can be merged since it has been released.

**Pull request merged**